### PR TITLE
Fix missing help page sections

### LINKED ISSUE #25: Missing help page sections
AskTheEU is missing some of the [required help sections](http://alaveteli.org/docs/customising/themes/#customising-the-help-pages).

#### Wrong place

- `/help/credits#helpus`  
AskTheEU has a custom [help out page](http://www.asktheeu.org/en/help/help_out). We can either merge this content back in to `/help/credits`, or change the link

#### Don't exist
- `/help/requesting#quickly_response`  
Goes to "How long do I have to submit an appeal", rather than "How quickly will I get a response?". There's no section for how long the authorities have to respond

- `/help/requesting#missing_body`  
There's no section for missing bodies

### PR DESCRIPTION
**Pull request by lizconlan**
A first attempt, may have missed some things.

Fixes #25

### CODE REVIEW

File: lib/views/help/requesting.html.erb [hunk 1]
```diff
@@ -43,17 +43,19 @@
...
   </ul>
 
+  <a name="missing_body"></a>
   <h2 id="finding_bodies">To which EU body should I make a request?</h2>
   <p>AsktheEU.org has <a href="http://www.asktheeu.org/en/body/list/all">a list of all of the EU's institutions, bodies, offices and agencies</a>.</p>
   <ul>
      <li>Browse or search AsktheEU.org looking for similar requests to yours.</li>
      <li>Browse and read the descriptions of each public body to see what they do, and/or check the body’s "home page".</li>
      <li>If you are still not sure, please contact us for help.</li>
+     <li>Please contact us if you think AsktheEU.org is missing an EU institution that comes under EU transparency rules</li>
```
Comment (garethrees) [inline lib/views/help/requesting.html.erb]: "Contact us" should link to the contact us page.

**Comment by garethrees** [comment]
Looks good, but doesn't cover the helpus issue:

> #### Wrong place
>
> - `/help/credits#helpus`  
> AskTheEU has a custom [help out page](http://www.asktheeu.org/en/help/help_out). We can either merge this content back in to `/help/credits`, or change the link

I _think_ you should be able to add a redirect in the custom routes file; something like `//help/credits#helpus => /help/help_out`. Obviously need to take the locale in to account, and don't really know if you can redirect based on anchor. If not will have to override templates that use something like `help_credits_path(:anchor => 'helpus')` and replace with the custom named route.

**Comment by lizconlan** [comment]
> Looks good, but doesn't cover the helpus issue:
> 
>> #### Wrong place
>> 
>> * `/help/credits#helpus`  
>>  AskTheEU has a custom [help out page](http://www.asktheeu.org/en/help/help_out). We can either merge this content back in to `/help/credits`, or change the link

Yeah, that was one of the bits which confused me originally

> I _think_ you should be able to add a redirect in the custom routes file; something like `//help/credits#helpus => /help/help_out`. Obviously need to take the locale in to account, and don't really know if you can redirect based on anchor.

Apparently not. Plus in trying to get it working, I've realised it's not actually linked from anywhere so now I have 2 problems. New sidebar link?

> If not will have to override templates that use something like help_credits_path(:anchor => 'helpus') and replace with the custom named route.

Looking at that next ^

**Comment by garethrees** [comment]
>  I've realised it's not actually linked from anywhere

Looks like its linked to from these places:

- https://github.com/mysociety/alaveteli/blob/3302a4c8c8a5202f63754acef603c479490bb0a4/app/controllers/request_game_controller.rb#L29
- https://github.com/mysociety/alaveteli/blob/3cacde6e382c4b8bc21e4503639a4f45e5e9f737/public/down.default.html#L43

**Comment by lizconlan** [comment]
> it's not actually linked from anywhere

Correction: it's linked from the sidebar in all 3 languages using the custom `help_help_out_path` link

**Comment by lizconlan** [comment]
Could not make it override the default `/help/credits#helpus` link (there might be a way involving hijacking the helper for `help_credits_path` to do something different when passed `:anchor => "help_us"` but I couldn't get it to work, and it wouldn't have solved the down page anyway.

Reluctantly overriding the core code.

**Pull request closed without merging**